charts/series.go

```diff
@@ -36,6 +36,7 @@ type SingleSeries struct {
 	Step         bool `json:"step,omitempty"`
 	Smooth       bool `json:"smooth,omitempty"`
 	ConnectNulls bool `json:"connectNulls,omitempty"`
+	ShowSymbol   bool `json:"showSymbol"`
```

Comment:
ShowSymbol bool `json:"showSymbol,omitempty"`?

Reply:
The default echart value is true. With `omitempty`, you can't set showSymbol to false from golang, as json ignores it while unmarshaling.

Reply:
Got

Reply:
The problem is that without specify `ShowSymbol`, we render no markers, which is not echarts' default behavior. Might have to set some boolean option to its echarts' default value before sending the json.

Reply:
It's a known problem of golang, might consider using `*bool` for some options.